build.sbt

```diff
@@ -598,9 +598,16 @@ lazy val scala_libraries_persistence =
         doobieCore,
         doobiePGDep,
         "org.reactivemongo" %% "reactivemongo" % reactiveMongo,
-        "org.reactivemongo" %% "reactivemongo-akkastream" % reactiveMongo,
+        "org.reactivemongo" %% "reactivemongo-akkastream" % reactiveMongo exclude("org.scala-lang.modules", "scala-parser-combinators_2.13"),
         "de.flapdoodle.embed" % "de.flapdoodle.embed.mongo" % embedMongoVersion % IntegrationTest,
-        logback
+        logback,
+        "com.typesafe.slick" %% "slick-hikaricp" % slickVersion,
+        "org.postgresql" % "postgresql" % "42.7.4",
```

Comment:
let's use variables for version numbers - follow same pattern as for other packages